gen/geometry/Pose2d.yml

```diff
@@ -7,7 +7,6 @@ functions:
     ignore: true
 classes:
   Pose2d:
-    shared_ptr: true
```

Comment:
shared_ptr defaults to true, so you should probably set it to false instead of deleting it...